src/lib/registration/registration.rb

```diff
@@ -164,6 +165,25 @@ def migration_products(installed_products)
       migrations
     end
 
+    # Get the list of updates for a given product
+    #
+    # @param [Hash]           Product. If nil the base product will be used.
```

Comment:
if it is documented elsewhere like in remote_product method, then at least please add `@see` to know where to look